src/index.html

```diff
@@ -5,9 +5,317 @@
+    <header id="home" class="header">
+      <div class="header__top">
+        <div class="logo header__logo"></div>
+        <div class="header__phone">
+          <a class="header__tel" href="tel:+1 234 555-55-55">
+            +1 234 555-55-55
+          </a>
+        </div>
+        <div class="header__action">
+          <a class="header__link" href="tel:+1 234 555-55-55">
+            Call to order
+          </a>
+        </div>
+      </div>
+
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item nav__item_active">
+            <a href="#home" class="nav__link-active">
+              Home
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#info" class="nav__link">
+              About Us
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gallery" class="nav__link">
+              Gallery
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#consultation" class="nav__link">
+              Consultation
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#testimonials" class="nav__link">
+              Testimonials
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#contact" class="nav__link">
+              Contacts
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section id="info" class="section">
+        <div class="info">
+          <h1 class="info__heading">
+              Miami <strong>best</strong> Real estate
+            <a href="#" class="link">
+              here
+            </a>
+          </h1>
+          <div class="info__text">
+            <p class="info__paragraph-top"><strong>Over 50 years experience in love.</strong><br>Mazel and VA Tsukkerman in his
+              "Analysis of musical works." Pointillism, which originated in the music of the early
+              twentieth century, microforms, found a distant historical.
+            </p>
+            <p class="info__paragraph-bottom">The market leader in real estate since 2003.<br>Awards Real Estate Company, 2010, 2011, 2013.
+              <br>More than 200 satisfied customers premium segment.<br>We focus only on new buildings and
+              high-level doskanalno know this area better than other specialists.
+              <br>Working directly with developers - so have more useful information about the profitable deals.
+            </p>
+            <p>
+              <strong>We will never be late!<br>Guaranteed secure transaction - 100%.</strong>
+            </p>
+          </div>
+        </div>
+      </section>
+      <section id="gallery" class="section">
+        <h2 class="heading section__heading">
+          Gallery
+          <span class="heading__crown"></span>
+        </h2>
+        <div class="gallery">
+          <div class="photos photos__wide_left">
```

Comment:
вот так будет правильно
```suggestion
          <div class="photos photos_wide-left">
```
или так
```suggestion
          <div class="photos photos--wide-left">
```